app/javascript/pages/QuestionIndex.js

```diff
@@ -2,9 +2,20 @@ import React, { Component } from 'react'
 
 export default class QuestionIndex extends Component {
     render() {
+        console.log("current user:", this.props.user)
```

Comment:
Please remove this console.log unless you still need it.

Reply:
HI Zan, sorry but yes I would really like to keep the console.logs while we're still developing if that's okay?